benchmarks/serde-benchmarks/README.md

```diff
@@ -0,0 +1,36 @@
+# Serde Benchmarks
+
+This project contains micro benchmarks for the serialization implementation(s).
+
+## Testing
+
+```sh
+./gradlew :runtime:serde:serde-benchmarks:jvmBenchmark
+```
+
+Baseline `0.4.0-alpha`
```

Comment:
nit

for convenience in reproducability maybe note the commit hash from which these baseline measurements since obviously this wasn't run on the commit of the alpha release.

Reply:
it wasn't but it also shouldn't matter we haven't made changes to serde since then